config/default/manager_config_patch.yaml

```diff
@@ -1,20 +0,0 @@
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: controller-manager
-  namespace: system
```

Comment:
could you please remove the lines (even though they are just comments) from the kustomize.yaml file? https://github.com/codeready-toolchain/member-operator/blob/ec800b33605d265754c7791b6345cc229f62a27e/config/default/kustomization.yaml#L28-L30

Reply:
thanks, done in [6f7182b](https://github.com/codeready-toolchain/member-operator/pull/465/commits/6f7182bd096ca266757a9725534e9969dcf9ba2a)